tensorflow_addons/utils/test_utils.py

```diff
@@ -177,6 +177,23 @@ def maybe_run_functions_eagerly(request):
     request.addfinalizer(finalizer)
 
 
+@pytest.fixture(scope="function", params=["CPU", "GPU"])
+def cpu_and_gpu(request):
+    if request.param == "CPU":
+        with tf.device("/device:CPU:0"):
+            yield
+    else:
+        if not tf.test.is_gpu_available():
+            pytest.skip("GPU is not available.")
+        with tf.device("/device:GPU:0"):
+            yield
+
+
+@pytest.fixture(scope="function", params=["channels_first", "channels_last"])
+def data_format(request):
```

Comment:
Ugh, it's a bit unfortunate that `data_format` is the tf / keras terminology for this. MXNet's `layout` or Caffe2's `order` better describe the variable IMO.

No sense in breaking what's established, just wanted to complain :P

Reply:
Haha, I never noticed. We just have to wait tensorflow 3.0 to change it :p